src/components/Map/Map.tsx

```diff
@@ -0,0 +1,119 @@
+import UrlIcon from 'leaflet/dist/images/marker-icon.png';
+import RetinaUrlIcon from 'leaflet/dist/images/marker-icon-2x.png';
+import ShadowUrlIcon from 'leaflet/dist/images/marker-shadow.png';
+import { icon } from 'leaflet';
+import {
+  AttributionControl,
+  MapContainer,
+  Marker,
+  TileLayer,
+  useMapEvents,
+} from 'react-leaflet';
+import React from 'react';
+
+import classes from './Map.module.css';
+import 'leaflet/dist/leaflet.css';
+
+// Default is center of Norway
+const DefaultCenter: [number, number] = [64.888996, 12.8186054];
+const DefaultZoom = 4;
+
+// Default map layers from Kartverket
+const DefaultMapLayers = [
+  {
+    url: 'https://opencache.statkart.no/gatekeeper/gk/gk.open_gmaps?layers=europa_forenklet&zoom={z}&x={x}&y={y}',
+    attribution: 'Data © <a href="http://www.kartverket.no/">Kartverket</a>',
+  },
+  {
+    url: 'https://opencache.statkart.no/gatekeeper/gk/gk.open_gmaps?layers=norgeskart_bakgrunn2&zoom={z}&x={x}&y={y}',
+    attribution: 'Data © <a href="http://www.kartverket.no/">Kartverket</a>',
+  },
+];
+
+export interface MapProps {
+  readOnly: boolean;
+  layers?: {
+    url: string;
+    attribution?: string;
+    subdomains?: string[];
+  }[];
+  center?: [number, number];
+  zoom?: number;
+  marker?: [number, number];
```

Comment:
I tried setting this in Storybook, but it didn't seem to do anything?
Maybe the prop can be a bit more descriptive: `markerLocation` perhaps?

Reply:
Fixed bug when setting marker location from Storybook. And also renamed `marker` to `markerLocation`